### PR TITLE
Update Watchers 

### PR DESCRIPTION
**Pull request by xellos00**
- Added
   - README.md
   - New Script to get valoperAddr(hex)
   - go_test

- Updated
   - Flag name alive vatz official plugins
   - resolve go lint error
--- 

Test Run
```
 dongyookang DK 🦄   ~/ffplay/GolandProjects/xellos/vatz  ➦ 2dc5550
 ./vatz plugin start --plugin watcher_cosmos --args "-valoperAddr=C0C1A472F5E49FB97C74788B015DE209DCB42046 -rpcURI=https://rpc.xion-testnet-1.burnt.com:443" --log watcher.logs
2024-07-19T22:29:17-05:00 INF Start plugin watcher_cosmos -valoperAddr=C0C1A472F5E49FB97C74788B015DE209DCB42046 -rpcURI=https://rpc.xion-testnet-1.burnt.com:443 module=plugin
2024-07-19T22:29:17-05:00 INF Plugin watcher_cosmos is successfully started. module=plugin
```

```
2024-07-19T21:42:14-05:00 INF Execute module=grpc
2024-07-19T21:42:14-05:00 DBG The validator is signing on the block successfully. module="plugin > watcher_cosmos"
2024-07-19T21:42:23-05:00 INF Execute module=grpc
2024-07-19T21:42:23-05:00 DBG The validator is signing on the block successfully. module="plugin > watcher_cosmos"
2024-07-19T21:42:32-05:00 INF Execute module=grpc
2024-07-19T21:42:32-05:00 DBG The validator is signing on the block successfully. module="plugin > watcher_cosmos"
2024-07-19T21:42:41-05:00 INF Execute module=grpc
2024-07-19T21:42:41-05:00 DBG The validator is signing on the block successfully. module="plugin > watcher_cosmos"
2024-07-19T21:42:50-05:00 INF Execute module=grpc
2024-07-19T21:42:50-05:00 DBG The validator is not signing the block. count: 1 module=plugin
2024-07-19T21:42:50-05:00 DBG  module="plugin > watcher_cosmos"
2024-07-19T21:42:59-05:00 INF Execute module=grpc
2024-07-19T21:42:59-05:00 DBG The validator is not signing the block. count: 2 module=plugin
2024-07-19T21:42:59-05:00 DBG  module="plugin > watcher_cosmos"
2024-07-19T21:43:08-05:00 INF Execute module=grpc
2024-07-19T21:43:08-05:00 DBG The validator is not signing the block. count: 3 module=plugin
2024-07-19T21:43:08-05:00 DBG The validator has failed to sign the block more than 3 times. module="plugin > watcher_cosmos"
2024-07-19T21:43:17-05:00 INF Execute module=grpc
2024-07-19T21:43:17-05:00 DBG The validator is not signing the block. count: 4 module=plugin
2024-07-19T21:43:17-05:00 DBG The validator has failed to sign the block more than 4 times. module="plugin > watcher_cosmos"
2024-07-19T21:43:26-05:00 INF Execute module=grpc
2024-07-19T21:43:26-05:00 DBG The validator is not signing the block. count: 5 module=plugin
2024-07-19T21:43:26-05:00 DBG The validator has failed to sign the block more than 5 times. module="plugin > watcher_cosmos"
2024-07-19T21:43:35-05:00 INF Execute module=grpc
2024-07-19T21:43:35-05:00 DBG The validator is not signing the block. count: 6 module=plugin
2024-07-19T21:43:35-05:00 DBG The validator has failed to sign the block more than 6 times. module="plugin > watcher_cosmos"
2024-07-19T21:43:44-05:00 INF Execute module=grpc
2024-07-19T21:43:44-05:00 DBG The validator is not signing the block. count: 7 module=plugin
2024-07-19T21:43:44-05:00 DBG The validator has failed to sign the block more than 7 times. module="plugin > watcher_cosmos"
2024-07-19T21:43:53-05:00 INF Execute module=grpc
2024-07-19T21:43:53-05:00 DBG The validator is not signing the block. count: 8 module=plugin
2024-07-19T21:43:53-05:00 DBG The validator has failed to sign the block more than 8 times. module="plugin > watcher_cosmos"
2024-07-19T21:44:02-05:00 INF Execute module=grpc
2024-07-19T21:44:02-05:00 DBG The validator is not signing the block. count: 9 module=plugin
2024-07-19T21:44:02-05:00 DBG The validator has failed to sign the block more than 9 times. module="plugin > watcher_cosmos"
2024-07-19T21:44:11-05:00 INF Execute module=grpc
2024-07-19T21:44:11-05:00 DBG The validator is signing on the block successfully. module="plugin > watcher_cosmos"
```

```
 make test
go test ./...
?       github.com/dsrvlabs/vatz-plugin-watcher-cosmos/plugins/watcher_cosmos   [no test files]
ok      github.com/dsrvlabs/vatz-plugin-watcher-cosmos/rpc      0.491s

```

---

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,103 @@
...
+> Please make sure [Vatz](https://github.com/dsrvlabs/vatz) is running with proper configuration. [Vatz Installation Guide](https://github.com/dsrvlabs/vatz/blob/main/docs/installation.md)
+
+### Install Plugins
+- Install with source
+```
+$ git clone https://github.com/dsrvlabs/vatz-plugin-watcher-cosmos.git
+$ cd vatz-plugin-watcher-cosmos
+$ make install
+```
+-
```
Comment (kim201212) [inline README.md]: Typo, Please fix it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,103 @@
...
+Enter the RPC address: http://127.0.0.1:26657
+Enter the endpoint: http://127.0.0.1:26657
+Enter the valoper address: osmovaloper1tt8dczjk62pnwwanm99rq2kw25r7vdgtxy5jym
+You have entered the following details:
+RPC address: http://127.0.0.1:26657
+Endpoint: http://127.0.0.1:26657
+Valoper address: osmovaloper1tt8dczjk62pnwwanm99rq2kw25r7vdgtxy5jym
+Are the entered values correct? Do you wish to proceed? (y/n): Y
+Validator address: cosmosvaloper1...
+```
```
Comment (kim201212) [inline README.md]: It differs from the script in this commit.
Which script is correct?

Comment (xellos00) [reply README.md]: > It differs from the script in this commit. Which script is correct?
@kim201212 
Readme.me has updated, yeah I made some modification and forgot to updated in readme.md. 
Please, check again.

File: README.md (hunk 1 above)
Comment (xellos00) [reply README.md]: Updated!!

**Review by kim201212 — APPROVED** [review]
LGTM!

**Comment by meetrick** [comment]
@xellos00 
Could you please check if get_veloper_addre_hex.sh is working correctly?

**Review by meetrick — APPROVED** [review]
LGTM

**Pull request merged**